server.js

```diff
@@ -204,6 +204,18 @@ app.get('/:region?/global-content/programmes/:country/:slug', (req, res) => {
     res.redirect(301, `${locale}/funding/programmes/${req.params.slug}`);
 });
 
+/**
+ * Handle migrated A Better Start child pages as wildcards
+ * (eg. redirect them to /funding/publications/a-better-start/<slug>)
+ */
+app.get('/funding/strategic-investments/a-better-start/:slug', (req, res) => {
+    const locale = req.params.region === 'welsh' ? '/welsh' : '';
```

Comment:
I'm missing something I don't see a `region` property in the params at all? Should this be using localise instead?

Reply:
Oh lol, that's from me copy-pasting the bit above. Will fix!